chainer/links/connection/highway.py

```diff
@@ -0,0 +1,71 @@
+from chainer.functions.activation import relu
+from chainer.functions.activation import sigmoid
+from chainer import link
+from chainer.links.connection import linear
+
+
+class Highway(link.Chain):
+
+    """Highway module.
+
+    Network allowing unimpeded information flow across layer on information
+    flow.
+    It applies two different gate. One gate is transform gate, and the other
+    is carry gate. Carry gate enable to propagete large gradient value to
+    previous layer.
+
+    The output is sum of linear multiplied transform gate, and input array
+    multiplied carry gate.
```

Comment:
The output is a sum of the output of a fully-connected layer with the specified activation function multiplied by the transform gate, and the input array multiplied by the carry gate.

(Maybe this corrected sentence is too long, so it would be better to split it into multiple sentences.)